code/modules/client/preference/loadout/loadout_uniform.dm

```diff
@@ -0,0 +1,102 @@
+// Uniform slot
+/datum/gear/uniform
+	subtype_path = /datum/gear/uniform
+	slot = slot_w_uniform
+	sort_category = "Uniforms and Casual Dress"
+
+/datum/gear/uniform/skirt
+	display_name = "skirt"
+
+/datum/gear/uniform/skirt/New()
+	..()
+	var/list/jumpskirts = list()
+	for(var/jump in subtypesof(/obj/item/clothing/under1))
+		if(findtext(path2text(jump), "skirt"))
+	gear_tweaks += new/datum/gear_tweak/path(sortAssoc(skirt_list))
```

Comment:
oh i see it now, you made var/list/jumpskirts, but used skirt_list instead, which is not defined